src/main/java/touro/peg/TriangleBoard.java

```diff
@@ -36,4 +41,21 @@ public String toString() {
                 intPeg(8), intPeg(9), intPeg(10), intPeg(11),
                 intPeg(12), intPeg(13), intPeg(14));
     }
+
+    public boolean isWin() {
+        int numTrues = 0;
+
+        for (boolean peg : pegs)
+        {
+            if (peg) {
+                numTrues++;
+            }
+        }
+        return numTrues == 1;
+    }
```

Comment:
Do this without "numTrues"

Reply:
How would I do that? Does it require a library or is it a built-in Java feature?